apps/web/src/AppRoutes.tsx

```diff
@@ -58,20 +58,26 @@ import {
   WorkflowsTestPage,
 } from './studio/components/workflows';
 import { WorkflowsStepEditorPageV2 } from './pages/templates/editor_v2/TemplateStepEditorV2';
+import { IS_CLERK_AUTH_ENABLED } from './config/index';
+import { ClerkRoutes } from './ee/clerk/ClerkRoutes';
 
 export const AppRoutes = () => {
   const isImprovedOnboardingEnabled = useFeatureFlag(FeatureFlagsKeysEnum.IS_IMPROVED_ONBOARDING_ENABLED);
 
   return (
     <Routes>
-      <Route element={<PublicPageLayout />}>
-        <Route path={ROUTES.AUTH_SIGNUP} element={<SignUpPage />} />
-        <Route path={ROUTES.AUTH_LOGIN} element={<LoginPage />} />
-        <Route path={ROUTES.AUTH_RESET_REQUEST} element={<PasswordResetPage />} />
-        <Route path={ROUTES.AUTH_RESET_TOKEN} element={<PasswordResetPage />} />
-        <Route path={ROUTES.AUTH_INVITATION_TOKEN} element={<InvitationPage />} />
-        <Route path={ROUTES.AUTH_APPLICATION} element={<QuestionnairePage />} />
-      </Route>
+      {!IS_CLERK_AUTH_ENABLED ? (
+        <Route element={<PublicPageLayout />}>
+          <Route path={ROUTES.AUTH_SIGNUP} element={<SignUpPage />} />
```

Comment:
The `/auth/login` URL is used and often times hardcoded on multiple places outside of web - I would rather not do this change as part of this PR since it would increase the scope and it would be harder to track the issues related to this path change.

Reply:
Do you have these places in mind? Are you talking about the marketing site? The CLI part is gone.

Reply:
I see. Let's leave it for this PR and we will address it separately.